os/Makefile.unix

```diff
@@ -441,7 +441,7 @@ include LibTargets.mk
 # is an archive.  Exactly what is performed during pass1 or what it generates
 # is unknown to this makefile unless CONFIG_PASS1_OBJECT is defined.
 
-PASS1_BUILDIR = $(BOARD_DIR)/userspace
+PASS1_BUILDIR = $(BOARD_DIR)/kernel
```

Comment:
This change is not required since we are presently using FLAT build. 
PASS1 and PASS2 is used only for PROTECTED build. Even in that case, PASS1_BUILDIR has to be userspace only.